sycl/doc/extensions/SPIRV/SPV_INTEL_long_constant_composite.asciidoc

```diff
@@ -0,0 +1,226 @@
+by one *OpTypeStruct* instruction, i.e. if number of members of the
+_Result type_ is greater than maximum possible WordCount, the remaining _member
+types_ are specified by following *OpTypeStructContinuedINTEL* instructions.
+
+In section 3.37.7. Constant-Creation Instructions, add the new instructions
+[cols="3", width="100%"]
+|=====
+2+^|*OpConstantCompositeContinuedINTEL* +
+
+Continue specifying an *OpConstantComposite* instruction with number of
+_Constituents_ greater than maximum possible WordCount.
+
+The previous instruction must be an *OpConstantComposite* or an
+*OpConstantCompositeContinuedINTEL* instruction.
+
+_Constituents_ follow the same rules as defined for _Constituents_ of
+*OpConstantComposite* instruction and specify members of a structure, or
+elements of an array, or components of a vector, or columns of a matrix.
+
+| Capability:
+*CapabilityLongConstantCompositeINTEL*
+
+| 2 + variable | 6091 | _<id>, <id>, ... Constituents_ |
+|=====
+[cols="3", width="100%"]
+|=====
+2+^|*OpSpecConstantCompositeContinuedINTEL* +
+
+Continue specifying an *OpSpecConstantComposite* instruction with number of
+_Constituents_ greater than maximum possible WordCount.
+
+The previous instruction must be an *OpSpecConstantComposite* or an
+*OpSpecConstantCompositeContinuedINTEL* instruction.
+
+_Constituents_ follow the same rules as defined for _Constituents_ of
+*OpSpecConstantComposite* instruction and specify members of a structure, or
+elements of an array, or components of a vector, or columns of a matrix.
+
+This instruction will be specialized to an *OpConstantCompositeContinuedINTEL*
+instruction.
+
+See <<Specialization, Specialization>>.
+
+| Capability:
+*CapabilityLongConstantCompositeINTEL*
+
+| 2 + variable | 6092 | _<id>, <id>, ... Constituents_ |
+|=====
+
+Modify the description of *OpConstantComposite* instruction, adding the
+following sentence to the end:
+In case if it is not possible to specify all the _Constituents_ by one
+*OpConstantComposite* instruction, i.e. if number of members of the
+_Result type_ and corresponding _Constituents_ is greater than maximum possible
+WordCount, the remaining _Constituents_ are specified by following
+*OpConstantCompositeContinuedINTEL* instructions.
+
+Modify the description of *OpSpecConstantComposite* instruction, adding the
+following sentence to the end:
+In case if it is not possible to specify all the _Constituents_ by one
+*OpSpecConstantComposite* instruction, i.e. if number of members of the
+_Result type_ and corresponding _Constituents_ is greater than maximum possible
+WordCount, the remaining _Constituents_ are specified by following
+*OpSpecConstantCompositeContinuedINTEL* instructions.
+
+In Section 3.37.8. Memory Instructions, modify the description of the
+*OpAccessChain* instruction, adding the following sentence to the end:
+In case _Base_ is an *OpConstantComposite* instruction followed by one or
```

Comment:
I think the same is applicable in case _Base_ is an **OpSpecConstantComposite**.

Reply:
Ok, mentioned that.